python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cantilever_beam_integration.py

```diff
@@ -73,12 +73,12 @@
 # We begin by getting the standard measure associated with the multivariate polynomial basis. We see that the range of the `Beta` distribution has been standardized into the [-1,1] interval. This is the same for the `Uniform` distribution and the second `Beta` distribution.
 
 # %%
-distributionMu = multivariateBasis.getMeasure()
-distributionMu
+distributionStandard = multivariateBasis.getMeasure()
+distributionStandard
```

Comment:
```suggestion
print(distributionStandard)
```
Otherwise the output is presented as normal text.